### PR TITLE
replace cell with cel on readme.md

### LINKED ISSUE #1: Correct all references to "cells"
There's always been inconsistencies in the spelling of the term for the basic image unit of a KiSS doll, and at first I was pretty sure it was supposed to be "cell". But looking at the original KiSS spec and considering the origin of the term is from animation (cel or celluloid), I'm now certain "cel" is the correct term and spelling. 

All the references to "cell" or "cells" should be changed to "cel" or "cels".

### PR DESCRIPTION
**Pull request by AlwinaO**
Fixes #1.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,5 @@
                     GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
+                       Verion 3, 29 June 2007
```
Comment (emhoracek) [inline LICENSE]: Please fix this to add the 's' back in!

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # smooch
 
 Smooch is a set of tools for viewing, converting, and eventually creating KiSS
-dolls. KiSS stands for Kisekae (Japanese for "dress up") Set System. 
+dolls. KiSS stands for Kisekae (Japanese for "dress up") Set System.
```
Comment (emhoracek) [inline README.md]: Trailing spaces!! Thank you for fixing this.

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ A typical KiSS doll: [Spark by Kimiki](http://otakuworld.com/kiss/dolls/pages/k/
 
 Check out Smooch's [GitHub wiki](https://github.com/emhoracek/smooch/wiki) if you want to learn more about KiSS.
 
-`cel2pnm` is a tool for converting KiSS cells (the individual graphics) to pnm
+`cel2pnm` is a tool for converting KiSS cels (the individual graphics) to pnm
```
Comment (emhoracek) [inline README.md]: 👍

**Comment by emhoracek** [comment]
Hi, @AlwinaO -- just checking in on this! Do you need any help with pushing new changes to the branch?

**Comment by AlwinaO** [comment]
Hello, please pardon the delay. I'll fix the License page.

**Comment by AlwinaO** [comment]
Fixed the LICENSE page by replacing Verion with Version.

**Comment by emhoracek** [comment]
This looks great! Thanks @AlwinaO!

**Pull request merged**